core/templates/vector.h

```diff
@@ -33,7 +33,7 @@
 
 /**
  * @class Vector
- * Vector container. Regular Vector Container. Use with care and for smaller arrays when possible. Use Vector for large arrays.
+ * Vector container. Simple, copy-on-write, array backed container.
```

Comment:
It isn't "array backed", this should probably suggest using `LocalVector` when appropriate instead

Reply:
Will do!